extensions/apps/settings/src/components/pages/notifications-preferences-option/app-setting.tsx

```diff
@@ -21,10 +21,20 @@ const AppSetting: React.FC<IAppSettingProps> = ({ title, description, isSelected
           id="checkbox"
           value="app-setting"
           name="app-setting"
-          isSelected={isSelected}
-          handleChange={onChange}
-          size="large"
-          customStyle="w-6 h-6"
+          checked={isSelected}
+          onCheckedChange={checked => {
+            const event = {
```

Comment:
this is too hacky. you are basically mocking an event here by providing only partial properties and passing it as it is a real event... I would adjust the onChange callback and adjust the parent